### PR TITLE
Add Tests for Common's post_id_helper() Method — ⌗118994

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**⌗118994**](http://central.tri.be/issues/118994)

* These tests are meant to help ensure that this _very-widely-used_ helper function has a rigid set of expected behaviors in response to passed values.
* This is valuable on its own, but we especially want this so that we can incorporate changes to this method like the ones in [this PR](https://github.com/moderntribe/tribe-common/pull/932) while ensuring it doesn't break anything elsewhere in our plugins. 

Test cases added:
 - [x] Positive ints
 - [X] Negative ints
 - [X] Zero
 - [X] Post object
 - [X] Null
 - [x] Strings
 - [X] Nothing (i.e. no argument specified for the function)

### CODE REVIEW

**Comment by bordoni** [comment]
Let's make sure we don't push commits with the composer development files here.

File: tests/wpunit/Tribe/Utils/Post_ID_HelperTest.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    /**
+     * @test When passing a WP_Post object, get the ID from it.
+     *
+     * @since TBD
+     *
+     * @param WP_Post $event_obj
+     *
+     * @dataProvider get_sample_event
+     */
+    public function it_should_return_post_ids_when_passed_post_objects( \WP_Post $event_obj ) {
```
Comment (bordoni) [inline tests/wpunit/Tribe/Utils/Post_ID_HelperTest.php]: also pass `use \WP_Post;`

Comment (ggwicz) [reply tests/wpunit/Tribe/Utils/Post_ID_HelperTest.php]: done in the new PR → https://github.com/moderntribe/tribe-common/pull/964

**Comment by ggwicz** [comment]
There's so much composer junk here that I just found it easier to make a new PR over here → https://github.com/moderntribe/tribe-common/pull/964

I am closing this one for now, please see that one.

**Pull request closed without merging**